tests/Chat/ChatSmokeTests.cs

```diff
@@ -470,4 +475,38 @@ public void SerializeChatMessageContentPartAsImageBytes(bool fromRawJson)
             Assert.That(additionalPropertyProperty.ValueKind, Is.EqualTo(JsonValueKind.True));
         }
     }
+
+    [Test]
+    [NonParallelizable]
+    public async Task HelloWorldChatWithTracingAndMetrics()
```

Comment:
Since this one is running against the live service, it doesn't fall in the "Smoke" category. Could you move this to ChatTests.cs file?

Reply:
This change will fix the CI pipeline.

Reply:
done, thanks!